client/src/components/Messaging/NewMessageForm.js

```diff
@@ -22,6 +24,12 @@ export const NewMessageForm = ({ handleSubmit, text, setText }) => {
         onChange={(e) => setText(e.target.value)}
         value={text}
       />
+      <div className={styles.imagePickerContainer}>
+        <label className={styles.imagePickerLabel} forhtml="file-upload">
```

Comment:
I think you wanted to use `htmlFor` here. I this case that also should not be needed as input is a child of label